templates/app/models/sequelize-model.hbs

```diff
@@ -11,6 +11,7 @@ module.exports = (sequelize, DataTypes) => {
       field: '{{field.nameColumn}}',{{/if}}{{#if field.primaryKey}}
       primaryKey: true,{{/if}}{{#if field.defaultValue}}
       defaultValue: Sequelize.literal('{{{field.defaultValue}}}'),{{/if}}
+      isReadOnly: {{field.isReadOnly}},
```

Comment:
Is `isReadOnly` a Sequelize field attribute?
https://sequelize.org/master/manual/model-basics.html#concept